package.json

```diff
@@ -4,6 +4,7 @@
   "private": true,
   "scripts": {
     "dev": "cross-env NODE_ENV=development next dev",
+    "docker-dev": "pnpm prisma push db && pnpm run dev",
```

Comment:
wrong command; its `pnpm prisma db push`

i did confirm if you checked once..

Reply:
https://github.com/SuperteamDAO/earn/blob/main/CONTRIBUTING.md#making-a-pull-request
also please check the option for allowing edits from maintainers

Reply:
> wrong command; its `pnpm prisma db push`
> 
> i did confirm if you checked once..

yes, there was typo.

Reply:
> https://github.com/SuperteamDAO/earn/blob/main/CONTRIBUTING.md#making-a-pull-request also please check the option for allowing edits from maintainers
sorry I am getting confused here if you are talking about 
the checkbox for allow edits from maintainers it is already checked.